scripts/provision.sh

```diff
@@ -222,6 +257,20 @@ generate_bldr_keys() {
   hab file upload "builder-api.default" "$(date +%s)" "/hab/cache/keys/${KEY_NAME}.box.key"
 }
 
+is_minio_migration_needed() {
+  installed_version=$(hab pkg path core/minio | cut -d '/' -f6)
+
+  response=$(curl -s "https://bldr.habitat.sh/v1/depot/channels/$BLDR_ORIGIN/$BLDR_CHANNEL/pkgs/builder-minio/latest")
+  minio_version_lookup=$(echo "$response" | jq -r '.deps[] | select(.origin == "core" and .name == "minio") | .version')
+  
+  echo minio_version_lookup $minio_version_lookup
+if [[ "${installed_version}" < "${BREAKING_MINIO_VERSION}" ]] && { [[ "${BREAKING_MINIO_VERSION}" < "${minio_version_lookup}" ]] || [ "${BREAKING_MINIO_VERSION}" = "${minio_version_lookup}" ]; }; then
```

Comment:
Why not `${BREAKING_MINIO_VERSION}" <= "${minio_version_lookup}`?

Reply:
<= is not working on timestamp comparison

Reply:
Yeah I just did a google search on this because it seems weird and found the same but what you can do is `! "${BREAKING_MINIO_VERSION}" > "${minio_version_lookup}"`.